### PR TITLE
Initial translation for Serbian language

### PR DESCRIPTION
**Pull request by dusanstojkovic**
Mobile translation - initial state - full translation
Web translation - initial state - full translation

merged upstream changes

### CODE REVIEW

File: mobile/translations.trm [hunk 1]
```diff
@@ -6,203 +6,252 @@
     <value lang="en" status="translated">Mon</value>
     <value lang="mk" status="translated">Пон</value>
     <value lang="de" status="translated">Mo</value>
+    <value lang="sr" status="initial">pon</value>
```
Comment (filipsardjoski) [inline mobile/translations.trm:9]: Please use capital first letters for the weekdays

File: mobile/translations.trm [hunk 2]
```diff
@@ -6,203 +6,252 @@
...
+    <value lang="sr" status="initial">Traži grad ili držav</value>
   </text>
   <text key="no_data_available">
     <context></context>
     <value lang="en" status="initial">No data available. Try again later.</value>
     <value lang="mk" status="initial">Нема достапни податоци. Обидете се повторно подоцна.</value>
     <value lang="de" status="translated">Keine Daten verfügbar. Versuchen Sie es später erneut.</value>
+    <value lang="sr" status="initial">Podaci su nedostupni. Probajte ponovo.</value>
+  </text>
+  <text key="disclaimer_message">
```
Comment (filipsardjoski) [inline mobile/translations.trm]: This looks like it's a duplicate of the one on line 236

File: mobile/translations.trm [hunk 3]
```diff
@@ -6,203 +6,252 @@
...
+    <value lang="mk" status="initial">srpski</value>
+    <value lang="de" status="initial">srpski</value>
+    <value lang="sr" status="initial">srpski</value>
   </text>
   <text key="english">
     <context></context>
     <value lang="en" status="initial">English</value>
     <value lang="mk" status="initial">English</value>
     <value lang="de" status="initial">English</value>
+    <value lang="de" status="initial">English</value>
```
Comment (filipsardjoski) [inline mobile/translations.trm]: This is an extra line

**Review by filipsardjoski — CHANGES_REQUESTED** [review]

File: mobile/translations.trm (hunk 1 above)
Comment (dusanstojkovic) [reply mobile/translations.trm:9]: In Serbian language, names of the days are lower case

File: mobile/translations.trm (hunk 2 above)
Comment (dusanstojkovic) [reply mobile/translations.trm]: should be fixed now

File: mobile/translations.trm (hunk 3 above)
Comment (dusanstojkovic) [reply mobile/translations.trm]: should be removed

**Comment by dusanstojkovic** [comment]
@sardzoski It should be ok now. 

I intentionally put status="initial" for the translations. Once confirmed how it looks in live app, I plan to change it to translated. I hope this status does not prevent displaying language in UI?

**Pull request merged**